ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-gas-limit/advanced-gas-fee-gas-limit.js

```diff
@@ -58,9 +58,9 @@ const AdvancedGasFeeGasLimit = () => {
   }
 
   return (
-    <Typography
-      tag={TypographyVariant.paragraph}
-      variant={TypographyVariant.H7}
+    <Text
+      tag={TextVariant.bodyMd}
+      variant={TextVariant.bodySm} as="h6"
```

Comment:
You still have a lint issue here that can be fixed with `yarn lint:fix`

Reply:
I'd love instead to help you get your setup working so that you could resolve some of the issues with the PR. The lint step of the CI process is still failing which makes reviewing the changes kind of a moot point right now. When You run `yarn lint:fix` what happens? Can you share some screenshots or terminal output?

Reply:
Sure.
Here are the screenshots
![Screenshot (9)](https://user-images.githubusercontent.com/125105825/233525256-9739f8f4-80ef-44ae-9f26-2d35658890d8.png)
![Screenshot (10)](https://user-images.githubusercontent.com/125105825/233525275-6b0252e7-2d0d-46ff-9b94-ea70c255798a.png)

Reply:
Also I am getting these problems in the file
![Screenshot (12)](https://user-images.githubusercontent.com/125105825/233525913-fca17e63-38ee-429e-8dfe-401e2dfada96.png)

Probably these are the lint issues you are talking about.

Reply:
In the last commit I have solved these CRLF lint issues.

@brad-decker , @georgewrmarshall 
Kindly check.